dask/array/core.py

```diff
@@ -1227,6 +1227,33 @@ def from_array(x, chunks, name=None, lock=False):
     return Array(merge({name: x}, dsk), name, chunks, dtype=x.dtype)
 
 
+def from_value(value, shape, dtype=None, name=None):
```

Comment:
`from_value` could probably use a better name.

Do we want to keep supporting `name=...` kwargs?  I don't think I've used this since the early days.